jaxb/appschema/build.gradle

```diff
@@ -0,0 +1,12 @@
+// increase the version number with every schema change!
+version = '2.9.3'
```

Comment:
Ops, I didn't get what the version number was for. Now I fixed it, thanks for pointing this out.